components/landing/highlights.json

```diff
@@ -0,0 +1,152 @@
+      "text": "Blocking of news websites and social media",
+      "report": "https://ooni.io/post/ethiopia-report/",
+      "explore": "/search?probe_cc=ET"
+    },
+    {
+      "countryCode": "UG",
+      "countryName": "Uganda",
+      "title": "Uganda 2016 general elections",
+      "text": "Blocking of social media",
+      "report": "https://ooni.io/post/uganda-social-media-blocked/",
+      "explore": "/search?probe_cc=UG"
+    }
+  ],
+  "media": [
+    {
+      "countryCode": "EG",
+      "countryName": "Egypt",
+      "title": "Egypt",
+      "text": "Blocking of hundreds of media websites",
+      "report": "https://ooni.io/post/egypt-internet-censorship/",
+      "explore": "/search?probe_cc=EG"
+    },
+    {
+      "countryCode": "SS",
+      "countryName": "South Sudan",
+      "title": "South Sudan",
+      "text": "Blocking of foreign-based media accused of hostile reporting against the government",
+      "report": "https://ooni.io/post/south-sudan-censorship/#blocked-websites",
+      "explore": "/search?probe_cc=SS"
+    },
+    {
+      "countryCode": "VE",
+      "countryName": "Venezuela",
+      "title": "Venezuela",
+      "text": "Blocking of independent media websites amid economic and political crisis",
+      "report": "https://ooni.io/post/venezuela-internet-censorship/#media",
+      "explore": "/search?probe_cc=VE"
+    },
+    {
+      "countryCode": "MY",
+      "countryName": "Malaysia",
+      "title": "Malaysia",
+      "text": "Blocking of media websites that covered the 1MDB scandal",
+      "report": "https://ooni.io/post/malaysia-report/#news-media",
+      "explore": "/search?probe_cc=MY"
+    },
+    {
+      "countryCode": "IR",
+      "countryName": "Iran",
+      "title": "Iran",
+      "text": "Blocking of at least 121 news outlets",
+      "report": "https://ooni.io/post/iran-internet-censorship/#news-websites",
+      "explore": "/search?probe_cc=IR"
+    }
+  ],
+  "lgbtqi": [
+    {
+      "countryCode": "ID",
+      "countryName": "Indonesia",
+      "title": "Indonesia",
+      "text": "",
+      "report": "https://ooni.io/post/indonesia-internet-censorship/#lgbt",
+      "explore": "/search?probe_cc=ID"
+    },
+    {
+      "countryCode": "IR",
+      "countryName": "Iran",
+      "title": "Iran",
+      "text": "",
+      "report": "https://ooni.io/post/iran-internet-censorship/#human-rights-issues",
+      "explore": "/search?probe_cc=IR"
+    },
+    {
+      "countryCode": "ET",
+      "countryName": "Ethiopia",
+      "title": "Ethiopia",
+      "text": "",
+      "report": "https://ooni.io/post/ethiopia-report/#lgbti-websites",
+      "explore": "/search?probe_cc=ET"
+    },
+    {
+      "countryCode": "MY",
+      "countryName": "Malaysia",
+      "title": "Malaysia",
+      "text": "",
+      "report": "https://sinarproject.org/digital-rights/updates/online-lgbt-censorship-malaysia",
+      "explore": "/search?probe_cc=MY"
+    }
+  ]
+}
```

Comment:
Eventually we should place this stuff inside of contentful so that we can make it easy for @agrabeli to edit it and keep it up to date.